tasks/libs/junit_upload_core.py

```diff
@@ -51,6 +54,18 @@ def junit_upload_from_tgz(junit_tgz, codeowners_path=".github/CODEOWNERS"):
     with open(codeowners_path) as f:
         codeowners = CodeOwners(f.read())
 
+    for test_file in TEST_OUTPUT_FILES:
+        if os.path.isfile(test_file):
+            with open(test_file) as f:
+                test_output = json.load(f)
+    else:
+        test_output = []
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏An indentation is needed here

Reply:
wow this is not a nit, this is a correction I need to perform, many thanks!!